templates/Apple/Applesauce WOZ.tcl

```diff
@@ -0,0 +1,177 @@
+# Apple/Applesauce WOZ.tcl
+# Binary format for Apple II disk images in the Applesauce WOZ format.
+# https://applesaucefdc.com/woz/reference2/
+# 2022 Aug 13 | chris-torrence | Initial implementation
+
+requires 0 "574F5A" ;# WOZ
+requires 4 "FF0A0D0A" ;# check bytes
+set woz [ascii 4]
+ascii 4
+entry "Signature" $woz 4 0
+entry "Checkbytes" "FF0A0D0A" 4 4
+# TODO: Should probably verify the crc32
```

Comment:
much effort to do this?

Reply:
yeah, it's probably 20-30 lines of code plus the 256-byte crc32 hex table. It's probably not worth it, as HexFiend shouldn't necessarily care if the crc32 is incorrect - it's up to the client using the binary file to throw an error. Presumably user's are just using HexFiend to view the contents. I think I'll just remove the comment.